edits

```diff
@@ -0,0 +1,7 @@
+@@ -2,5 +2,5 @@
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+> "Failure is not the problem, failing to try is"
+Use the left arrow to go back!
```

Comment:
Remember, you need to add a space before this line, otherwise it will run together with the quote.